setup/COSIGN.md

```diff
@@ -0,0 +1,101 @@
+# Using Connaisseur with Cosign signatures
+
+[Sigstore](https://sigstore.dev/) is a [Linux Foundation](https://linuxfoundation.org/) project that aims to provide public software signing and transparency to improve open source supply chain security. As part of the Sigstore project, [Cosign](https://github.com/sigstore/cosign) allows seamless container signing, verification and storage. You can read more about it [here](https://blog.sigstore.dev/cosign-signed-container-images-c1016862618?gi=95796f563c9f).
+
+As a first step, we have implemented the elementary function of verifying `cosign` generated signatures against the locally created corresponding public keys. We plan to expose further features of cosign in upcoming releases, so stay tuned!
+
+## Demo
+![](../img/co-nny-sign.gif)
+
+## Signing Container Images with Cosign
+
+Getting started with `cosign` is very well described in the [docs](https://github.com/sigstore/cosign). Please check there for detailed instructions. In short: After installation a keypair is generated via:
+
+```bash
+cosign generate-key-pair
+```
+
+You will be prompted to enter a password and a private (`cosign.key`) and public (`cosign.pub`) key are created. You can then use cosign to sign a container image using:
+
+```bash
+# Here, <IMAGE> is REPOSITORY/IMAGE_NAME:TAG
+cosign sign -key cosign.key <IMAGE>
+```
+
+You can then verify the created signature via:
+
+```bash
+cosign verify -key cosign.pub <IMAGE>
+```
+
+
+
+## Verification with Connaisseur
+
+Setting up Connaisseur (or short: `conny`) for `cosign` signatures only requires minor changes. In case of questions, please refer to the [default guide](README.md). In essence, you can just clone this repository:
```

Comment:
wouldn't name Connaisseur by it's nickname in the official docs

Reply:
actually a bigger fan of the nickname than the real name bc it is way too long and confuses people, but will do

Reply:
duly done :disappointed: